features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Listing/components/ImageGenerator/ApiThumb.jsx

```diff
@@ -124,7 +124,12 @@ class APIThumb extends Component {
     render() {
         const { classes, api, isAPIProduct } = this.props;
         const { isHover, loading } = this.state;
-        api.apiType = API.CONSTS.API;
+        if (api.type === 'APIPRODUCT') {
+            api.state = 'PUBLISHED';
```

Comment:
we can also include L134 into this if what `isAPIProduct` method does is to check whether `api.type` is `APIPRODUCT` or not.

Reply:
Fixed in 6f7533f434fe1ec30de4119eb74cd6d7f95880da